src/mlpack/methods/ann/layer/multihead_attention_impl.hpp

```diff
@@ -202,8 +203,9 @@ Backward(const MatType& /* input */,
   // The shape of gyTemp : (tgtSeqLen, embedDim, batchSize).
   // We need not split it into n heads now because this is the part when
   // output were concatenated from n heads.
-  CubeType gyTemp(const_cast<MatType&>(gy).memptr(), embedDim,
-      tgtSeqLen, batchSize, true, false);
+  CubeType gyTemp;
+  MakeAlias(gyTemp, gy, embedDim,
+      tgtSeqLen, batchSize, 0, false);
```

Comment:
```suggestion
  MakeAlias(gyTemp, gy, embedDim, tgtSeqLen, batchSize, 0, false);
```
This all fits on one line, so no need to wrap.